<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="23149" started-at="2018-08-06T17:02:51Z" finished-at="2018-08-06T17:03:14Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="23149" started-at="2018-08-06T17:02:51Z" finished-at="2018-08-06T17:03:14Z">
      <class name="jirazephyrpaxk.jira2">
        <test-method status="PASS" signature="zephyrtestingtwo()[pri:0, instance:jirazephyrpaxk.jira2@68c4039c]" name="zephyrtestingtwo" duration-ms="14848" started-at="2018-08-06T22:32:51Z" finished-at="2018-08-06T22:33:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- zephyrtestingtwo -->
      </class> <!-- jirazephyrpaxk.jira2 -->
      <class name="jirazephyrpaxk.jira">
        <test-method status="FAIL" signature="zephyrtesting()[pri:0, instance:jirazephyrpaxk.jira@27efef64]" name="zephyrtesting" duration-ms="8256" started-at="2018-08-06T22:33:05Z" finished-at="2018-08-06T22:33:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Google] but found [Testing]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Google] but found [Testing]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at jirazephyrpaxk.jira.zephyrtesting(jira.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- zephyrtesting -->
      </class> <!-- jirazephyrpaxk.jira -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
